build.gradle

```diff
@@ -130,7 +133,12 @@ subprojects {
     dependencies {
         minecraft "net.minecraftforge:forge:${forge_version}"
         library "org.slf4j:slf4j-api:2.0.16"
-
+        compileOnly("net.neoforged.fancymodloader:loader:2.0.20") {
```

Comment:
Both need to be in if (versionId >= 1_20_6) to not be included in older versions